authors/apps/tests/test_rating_article.py

```diff
@@ -0,0 +1,108 @@
+"""
+test module for rating an article
+"""
+from django.urls import reverse
+
+from authors.apps.tests import post_article
+from authors.apps.tests.base import BaseTestCase
+
+
+class TestRatingArticle(BaseTestCase):
+    """
+    test class to contain functions to test rating an article
+    """
+
+    def test_rate_your_article(self):
+        """
+        method to test rating your own article
+        :return:
+        """
+        self.authorize_user()
+        self.posting_article(post_article)
+
+        response = self.client.get(reverse("articles"), format='json')
+        data = response.json().get("articles")[0]
+        slug = data["slug"]
+        response1 = self.client.post(
+            reverse("rating", kwargs=dict(slug=slug)),
+            data={"article": {"score": 3}},
+            format="json"
+        )
+        self.assertEqual(response1.status_code, 400)
+        self.assertIn("errors", response1.json())
+
+    def test_rate_someones_article(self):
+        """
+        method to test rating someone's article
+        :return:
+        """
+        self.authorize_user()
+        self.posting_article(post_article)
+
+        self.authorize_user2()
+        response = self.client.get(reverse("articles"), format='json')
```

Comment:
Could you handle this get request while initialising an object in this class? Ideally, it is clumsy to have two responses in the same method. It is something I have noticed throughout this test suite.

Reply:
cool let me work on it